ext/B/B.pm

```diff
@@ -593,6 +593,18 @@ C<REFCNT> (corresponding to the C function C<SvREFCNT>).
 
 =item FLAGS
 
+=item is_bool
+
+Returns true if the SV is a boolean (PL_Yes or PL_No).
+
+=item is_bool_yes
+
+Returns true if the SV is a boolean and is PL_Yes.
```

Comment:
"... is a boolean and true". Most regular SVs that are boolean and true will not be exactly `PL_sv_yes`

Reply:
going to update it as suggested